src/translations/en.json

```diff
@@ -786,7 +786,31 @@
               "door": "Door",
               "garage_door": "Garage door",
               "window": "Window",
-              "opening": "Other"
+              "opening": "Other",
```

Comment:
Opening = Other ?

Reply:
🤷‍♂️  It was this way before haha. I guess opening is an interesting device class